### DISCUSSION TITLE
Available Information from the API

### DISCUSSION THREAD

**Discussion by maedball** [flobz/psa_car_controller#405] [General]
Hey Guys,

i am currently trying to get some information what is possible with the APIs from Stellantis.

Does someone know if it is possible on a PHEV to see how many KM have been driven with Gas and how many with Electric?

**Comment by PatrikNorrgard** [comment]
I don´t get any trips whatsoever, and no charging events. Battery-, gas-levels, charge control and pre-condition works.
Opel Grandland X Phev 2022.

**Comment by maedball** [comment]
Sorry I think I should have phrased it differently :-)

I have the Controller already working and running, my question is more if there is more available directly from stellantis which the controller is currently not providing.
